soft/jdk1.8.0_20/bin/java -Xmx32G -jar soft/gatk-3.6/GenomeAnalysisTK.jar -T VariantRecalibrator -R hg19/ucsc.hg19.fasta -input out.raw.vcf  -resource:mills,known=true,training=true,truth=true,prior=12.0 hg19/Mills_and_1000G_gold_standard.indels.hg19.sites.vcf -resource:dbsnp,known=true,training=false,truth=false,prior=2.0 hg19/dbsnp_138.hg19.vcf -an QD -an FS -an SOR -an MQRankSum -an ReadPosRankSum  -mode INDEL -nt 6 -tranche 100.0 -tranche 99.9 -tranche 99.0 -tranche 90.0 --maxGaussians 4 -recalFile out.recalibrate_INDEL.recal -tranchesFile out.recalibrate_INDEL.tranches -rscriptFile out.recalibrate_INDEL_plots.R && soft/jdk1.8.0_20/bin/java -Xmx32G -jar soft/gatk-3.6/GenomeAnalysisTK.jar -T ApplyRecalibration -R hg19/ucsc.hg19.fasta -input out.recalibrated_snps_raw_indels.vcf -mode INDEL --ts_filter_level 99.0 -recalFile out.recalibrate_INDEL.recal -tranchesFile out.recalibrate_INDEL.tranches -o out.recalibrated_variants.vcf
